inc/classes/Addons/Cloudflare/Cloudflare.php

```diff
@@ -0,0 +1,611 @@
+<?php
+namespace WP_Rocket\Addons\Cloudflare;
+
+use WP_Rocket\Admin\Options_Data;
+
```

Comment:
```
use Cloudflare\Api;
use Cloudflare\Zone\Cache;
use Cloudflare\Zone\PageRules;
use Cloudflare\Zone\Settings;
use Cloudflare\IPs;
```
Importing the namespaces here makes the code easier to read and update when we need to create the instances of the classes later on